### ISSUE TITLE
mega-linter-runner doesn't default to v6 for flavors

### ISSUE THREAD

**Issue by karbassi** [oxsecurity/megalinter#1596]
**Describe the bug**
After upgrading `mega-linter-runner` to v6, when running `mega-linter-runner` with the `--flavor javascript` flag, it defaults to v5.

**To Reproduce**

1. `npx mega-linter-runner --flavor javascript`

**Expected behavior**
Should use v6 since it's stable.

**Screenshots**

```sh
$ npx mega-linter-runner --flavor javascript
#######################################################################
MEGA-LINTER HAS A NEW V6 VERSION. Please upgrade to it by:
- Running the command at the root of your repo (requires node.js): npx mega-linter-runner --upgrade
#######################################################################
Pulling docker image megalinter/megalinter-javascript:v5 ...
INFO: this operation can be long during the first use of mega-linter-runner
The next runs, it will be immediate (thanks to docker cache !)
v5: Pulling from megalinter/megalinter-javascript
...
...
```

**Additional context**

Running `npx mega-linter-runner -r v6 -f javascript` fixes the problem.

**Comment by nvuillam** [comment]
@karbassi Thank you for reporting !
I found the source of the error, it will be in a new v6 path tomorrow :)
Meanwhile you find the good workaround !

**Comment by nvuillam** [comment]
@karbassi fixed in v6.0.3 :)
